storybookjs/storybook
# Add `--tag` option to `upgrade` script

Issue: N/A

## What I did

Allow passing `--tag` to the prerelease script to use a npm dist-tag.

## How to test

- Run `node path/to/cli upgrade --tag next` -- check it works
- Run `node path/to/cli upgrade --prerelease --tag next` -- check it errors

I'm not sure what we should do about not using the right version of the CLI to upgrade to later version of the packages. Did you have thoughts there @yannbf?
